package.json

```diff
@@ -24,15 +24,42 @@
     "dev": "node_modules/.bin/concurrently -k -n \"TypeScript,Node\" \"npm run build:watch\" \"npm run start\"",
     "lint": "node_modules/.bin/tslint 'src/**/*.{ts,tsx}'",
     "lint:fix": "npm run lint -- --fix",
-    "start": "node_modules/.bin/nodemon .",
+    "start": "node_modules/.bin/nodemon ./dist/server.js",
     "test": "node_modules/.bin/jest",
     "test:coverage": "node_modules/.bin/jest --coverage --runInBand --ci",
     "test:watch": "node_modules/.bin/jest --watch",
-    "typeorm": "ts-node ./node_modules/typeorm/cli.js --config ormconfig.ts",
+    "typeorm": "ts-node ./node_modules/typeorm/cli.js --config dist/config/orm.js",
     "postinstall": "test -f .env || cp -n .env.example .env"
   },
-  "dependencies": {},
-  "devDependencies": {},
+  "dependencies": {
+    "@hapi/joi": "^15.1.1",
+    "@pod-point/typescript": "^1.1.0",
+    "@sentry/node": "^5.9.0",
+    "express": "^4.17.1",
+    "express-async-errors": "^3.1.1",
+    "http-status-codes": "^1.3.2",
+    "mysql": "^2.17.1",
+    "typeorm": "^0.2.18"
+  },
+  "devDependencies": {
+    "@types/express": "^4.17.1",
+    "@types/faker": "^4.1.5",
+    "@types/hapi__joi": "^15.0.4",
+    "@types/jest": "^24.0.18",
+    "@types/node": "^12.7.3",
+    "@types/supertest": "^2.0.8",
+    "concurrently": "^4.1.2",
+    "faker": "^4.1.0",
+    "jest": "^24.9.0",
+    "nodemon": "^1.19.1",
+    "sqlite3": "^4.1.0",
+    "supertest": "^4.0.2",
+    "ts-jest": "^24.0.2",
+    "ts-node": "^8.3.0",
+    "tslint": "^5.19.0",
+    "tslint-eslint-rules": "^5.4.0",
+    "typescript": "^3.5.3"
+  },
```

Comment:
I purposely left all of this + the package-lock.json out so that latest versions were installed by running: https://github.com/Pod-Point/node-express-api/pull/2/files#diff-04c6e90faac2675aa89e2176d2eec7d8R10

Any new/different packages should be added there, can discuss when you're in tomorrow